packages/spec/src/locales.ts

```diff
@@ -221,7 +221,7 @@ export const localeCodeSchema = Z.string().refine((value) => localeCodes.include
   message: "Invalid locale code",
 });
 
-export const resolveLocaleCode = (value: LocaleCode): LocaleCodeFull => {
+export const resolveLocaleCode = (value: string): LocaleCodeFull => {
```

Comment:
since the function already handles case for an invalid code on line `239`, we can make the input param type as string

Reply:
Good catch! 🚀